utils/repository.go

```diff
@@ -15,13 +15,21 @@ func OwnerRepo(repoName string) (string, string, *errors.Error) {
 		return "", "", errors.New("parsing repository name: invalid number of parts")
 	}
 
+	errInvalidCharacters := errors.New("repository name contains invalid characters")
```

Comment:
put this near the top of the file as a `var` declaration and make it public (capitalized). this way other consumers can use it.

Reply:
should i also do  this for other errors in file ?